pyproject.toml

```diff
@@ -82,7 +82,7 @@ doc = [
     "sphinxcontrib-websupport==2.0.0",
     "sphinxemoji==0.3.1",
 ]
-viz = [
+graphics = [
```

Comment:
Don't you want to still leave it for backwards compatibility and deprecate it in the next minor release? Up to you..